### DISCUSSION TITLE
NoURLAvailableError, external url setup (e.g. DuckDNS)

### DISCUSSION THREAD

**Discussion by fwestenberg** [fwestenberg/reolink_dev#222] [Q&A]
Add this to your configuration.yaml:

```
homeassistant:
customize: !include customize.yaml
internal_url: https://internal_address:8123
external_url: https://duck_dns_alias.duckdns.org
```

At least internal_url should not be initial, in yaml mode or in your configuration:
![image](https://user-images.githubusercontent.com/47930023/134022634-07c2a8e3-3a05-4d05-a7da-021af3bdcf78.png)

**Comment by andriej** [comment]
https in internal_url? Is it or it will be supported soon?

**Reply by fwestenberg** [reply]
I already asked Reolink to fix this (or help us fix it). But no solution is offered yet. We really depend on this. I will post any updates here!

**Reply by andriej** [reply]
Is there any way in which community could help with that? 
I can see that somehow Reolink 'stopped' developing new firmwares for "older" products, usually there were new versions quarterly at max.
I didn't even manage to find way to contact them directly... ;-)

**Reply by fwestenberg** [reply]
I have a very kind contact at Reolink, some things are already solved so hopefully this also will be solved.

**Reply by andriej** [reply]
Hi - just quick follow-up: any news on the topic? :-)

**Reply by fwestenberg** [reply]
Nope. :(
